apps/framework-cli/src/cli/routines/ls.rs

```diff
@@ -157,3 +187,47 @@ fn augment_output_table(
         }
     }
 }
+
+async fn get_topics(project: &Project) -> HashSet<String> {
+    let regex_filter = Regex::new(r"_\d+_\d+$").unwrap();
```

Comment:
This feels very fragile.

Can we load the flows (including flow version syncs) and output that?

Reply:
Normal flow topics are just the data model topics and they work fine. I haven't been able to query DCM flow topics though. Would you happen to know why?

Reply:
This is the list from framework-internal-app before it's filtered. I supposed I could just remove __consume_offsets, but still not sure why DCM flow topics aren't showing.

PageViewEvent_0_0
UserActivity_0_0
ParsedActivity_0_0
MooseActivity_0_0
MooseActivity_0_1
MooseActivity_0_2
UserActivity_0_3
MooseActivityAugmented_0_4
MooseActivity_0_4
__consumer_offsets

Reply:
Maybe we can start there and then use the infrastructure map?

Reply:
They're named

UserActivity_0_0_UserActivity_0_1_input
UserActivity_0_0_UserActivity_0_1_output

Reply:
@callicles , sounds good, would be great to converge on one source of truth.

@phiSgr , I changed it to a blacklist approach. LMK if that works